app/src/main/java/com/vinh/dictionary_1/screen/worddetail/WordDetailViewModel.java

```diff
@@ -35,6 +39,13 @@ public void setPresenter(WordDetailContract.Presenter presenter) {
         mPresenter = presenter;
         mWordDetailWebViewClient = new WordDetailPresenter.WordDetailWebViewClient(mContext,
                 mPresenter.getDictRepository());
+        if (mPresenter.isWordBookmarked(mWord.getWord())) {
```

Comment:
Pls change to
boolean isBookbark = mPresenter.isWordBookmarked(mWord.getWord());
mWord.setBookmarked(isBookbark );
setBookmarkState(isBookbark );